{"name": "test_get_with_redis_error", "status": "passed", "description": "Test get met Redis error.", "attachments": [{"name": "log", "source": "1eb50e02-ddb9-4226-8a08-01172cd95118-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "78821b1c-ffd0-405b-9a3e-881ebc1836db-attachment.txt", "type": "text/plain"}], "start": 1753819436189, "stop": 1753819436190, "uuid": "dc339842-9f6e-4c63-b780-7560e956b803", "historyId": "4d3e7d69a3c6835174ca1110416245a6", "testCaseId": "4d3e7d69a3c6835174ca1110416245a6", "fullName": "tests.unit.core.test_redis_cache_coverage.TestRedisCacheCoverage#test_get_with_redis_error", "labels": [{"name": "parentSuite", "value": "tests.unit.core"}, {"name": "suite", "value": "test_redis_cache_coverage"}, {"name": "subSuite", "value": "TestRedisCacheCoverage"}, {"name": "host", "value": "Yannick-Mac-Gillavry-werk.local"}, {"name": "thread", "value": "90219-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.unit.core.test_redis_cache_coverage"}], "titlePath": ["tests", "unit", "core", "test_redis_cache_coverage.py", "TestRedisCacheCoverage"]}